lib/mbtiles.js

```diff
@@ -160,6 +171,10 @@ MBTiles.prototype.getTile = function(z, x, y, callback) {
 
     var sql = 'SELECT tile_data FROM tiles WHERE zoom_level = ? AND tile_column = ? AND tile_row = ?';
     var mbtiles = this;
+    if (process.env.BRIDGE_LOG_MAX_VTILE_BYTES_COMPRESSED) {
+        stats && stats.count++;
```

Comment:
Let's move this to within the `this._db.get` callback, so we can get the size of the tile with `row.tile_data.length`